app/assets/javascripts/app/form-validation.js

```diff
@@ -11,6 +11,29 @@ document.addEventListener('DOMContentLoaded', () => {
 
   if (forms.length !== 0) {
     [].forEach.call(forms, function(form) {
+      const buttons = form.querySelectorAll('input[type="submit"]');
+      form.addEventListener('submit', function() {
+        if (buttons.length !== 0) {
+          [].forEach.call(buttons, function (button) {
+            button.disabled = true;
+          });
+        }
+      }, false);
+      const elements = form.querySelectorAll('input');
```

Comment:
This looks pretty inefficient to me, and also ties to input rather than button, so you'll be looking at a lot of inputs unnecessary, and miss `button` elements. What do you think of removing any existing disabled attributed onload?

Reply:
it won't work because there is no onload fired when coming from the back button